### ISSUE TITLE
Version 2.0 (Beta and Stable) isn't working properly

### ISSUE THREAD

**Issue by JamesDDunn** [katzwebservices/Gravity-Forms-Mad-Mimi#6]
With the Stable version active, and feeds set up, when a form is submitted, I get the following error:

Fatal error: Call to undefined method MadMimi_SuperClass::listSubscribe() in /home/wmslfm/public_html/wp-content/plugins/gravity-forms-mad-mimi/madmimi.php on line 1171

I decided to try the Beta instead. However, when it is active, and I go to the section to set up the feeds (because no feeds show up - even the ones I set up in the Stable version), the field for the MadMimi lists does not even show up - no dropdown, no fill-in box, nothing. 

So basically, neither version will work for me. 

Any ideas?

**Comment by zackkatz** [comment]
This should be fixed in 2.0.1 - sorry for the problems.

**Comment by JamesDDunn** [comment]
No worries Zach. Any idea on when that will be ready? BTW, do you have a place to make a contribution for your efforts? I can probably "extract" some funds from a few clients to send your way. When this works, it rocks. :+1:

**Comment by zackkatz** [comment]
It's out now!

Thanks, donations are great :-)
